governance-exercises/Le Bateleur - Interpretation Exercise 01/Assessment of Ambiguity.md

```diff
@@ -0,0 +1,20 @@
+# Interpretation Exercise 01
+
+---
+
+## Is there ambiguity in these clauses?
+
+I assess these clauses as ambiguous/overbroad. In making this assessment, I am guided by analogous principles of statutory interpretation whereby ambiguity/overbreadth is determined not only by referencing the language at issue, but also the specific context in which that language appears and the broader context of the statutory scheme as a whole.  
+
+The following language is problematic:
+
+- "...when this relates to unintended consequences or mistakes in the affected documents." (GOV 12.1)
+- "In case of unintended consequences or mistakes in the Atlas and Scope Artifacts causing Maker Governance to function incorrectly..." (GOV 12.1.1)
+
+The most glaring deficiency is the phrase "when this relates to". This is an [omnibus term](https://www.law.cornell.edu/wex/omnibus_clause#:~:text=A%20contractual%20provision%20that%20operates,a%20last%20will%20and%20testament.) - an indeterminate catch-all (i.e., "relating to", pertaining to", "concerning") common to legal artifacts. Its purpose is to sweep under the legal instrument's purview events and conditions not actually specified in the instrument. This phrase enables an expansive reading of GOV 12 that poses a significant risk to Maker Governance.
```

Comment:
Today I learned. Good to know (but on reflection unsurprising) that this is a real thing that is used frequently and has its own term. 

Not something I'd explicitly picked up on before. But you're right that someone could argue that its intended as an omnibus term and therefore allows more or less anything.